swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -483,20 +395,22 @@ type Payment record {|
     decimal discounted;
     string paymentID;
     string status;
-|};
+};
 
-type ReservationRequest record {|
-    record {|
-        *Patient;
-        string cardNo;
-    |} patient;
+type PatientWithCardNo record {
```

Comment:
@MaryamZi So we are not going to have closed records in the complete source code as well right?

Reply:
No. What I meant is that when we say this is the generated code, it has to be the generated code. We can then close it and include a note and say we can make these closed records and give the reason.